w3c.json

```diff
@@ -0,0 +1,5 @@
+ {
+    "group":      109735
+,   "contacts":   ["dontcallmedom", "himorin"]
+,   "repo-type":  "homepage"
```

Comment:
hmm... It's not clear to me that the "homepage" classification is a good fit; also, it fills more of a CG repo than WG repo

Reply:
for WG->CG, I could agree. but I could not find better classification... (agree that this is not a kind of general "homepage", but not a "tool",,, so "others"??)

Reply:
Would "article" maybe be applicable (non-spec documents)?

Reply:
it's actually provided as [samples directory](https://immersive-web.github.io/webxr-samples/) but not as documentation. We might be possible to call source codes of samples as documentation,,, for some cases, but...